tests/test_strm_scheduler.py

```
@@ -0,0 +1,120 @@
+    assert job.result_path
+
+    out_path = Path(job.result_path)
+    assert out_path.exists()
+    data = out_path.read_bytes()
+    assert data == b"https://example.com/video.mp4\n"
+    assert b"\r\n" not in data
+
+
+def test_run_strm_marks_failed_on_invalid_url(tmp_path, monkeypatch):
+    scheduler = _setup_scheduler(tmp_path, monkeypatch)
+    monkeypatch.setattr(scheduler, "build_episode", lambda **kwargs: object())
```

Ruff (0.14.8)

Unused lambda argument: `kwargs`

(ARG005)